apps/desktop/src/lib/gitHost/gitHostFactory.ts

```diff
@@ -36,7 +37,7 @@ export class DefaultGitHostFactory implements GitHostFactory {
 				usePullRequestTemplate
 			});
 		}
-		if (source.includes(GITLAB_DOMAIN)) {
+		if (source.includes(GITLAB_DOMAIN) || resource.includes(GITLAB_SUB_DOMAIN)) {
```

Comment:
Thanks for contributing! :)

Would it also be a good idea to make it stricter like this?
```ts
if (source === GITLAB_DOMAIN || resource.startsWith(GITLAB_SUB_DOMAIN + ".")) {
```
The current code is incorrect in that e.g. `agitlab.com` would be recognised as GitLab.

Reply:
Makes sense